compiler-rt/test/tysan/lit.cfg.py

```diff
@@ -0,0 +1,161 @@
+# -*- Python -*-
+
+import os
+import platform
+import re
+
+import lit.formats
+
+# Get shlex.quote if available (added in 3.3), and fall back to pipes.quote if
+# it's not available.
+try:
+    import shlex
+
+    sh_quote = shlex.quote
+except:
+    import pipes
+
+    sh_quote = pipes.quote
+
+
+def get_required_attr(config, attr_name):
+    attr_value = getattr(config, attr_name, None)
+    if attr_value == None:
```

Comment:
`is None`

Reply:
Changed to check `if attr_value` and return value early.

Reply:
just confirming it is expected that empty string, or false, or something like that will now fail

Reply:
I've adjusted to return if not None, thanks